Packs/Base/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -0,0 +1,588 @@
+
+    pred_json[DOMAIN_AGE_KEY] = is_new_domain
+
+    score, verdict = get_verdict(pred_json, is_white_listed)
+
+    return create_dict_context(url, verdict, pred_json, score, is_white_listed, output_rasterize)
+
+
+def return_general_summary(results, tag="Summary"):
+    df_summary = pd.DataFrame()
+    df_summary['URL'] = [x.get('url') for x in results]
+    df_summary[KEY_FINAL_VERDICT] = [MAPPING_VERDICT_COLOR[x.get('verdict')] % x.get('verdict')
+                                     if x.get('verdict') in MAPPING_VERDICT_COLOR.keys()
+                                     else VERDICT_ERROR_COLOR % x.get('verdict') for x in results]
+    df_summary_json = df_summary.to_dict(orient='records')
+    return_entry = {
+        "Type": entryTypes["note"],
+        "ContentsFormat": formats['json'],
+        "HumanReadable": tableToMarkdown("Phishing prediction summary for URLs", df_summary_json,
+                                         headers=['URL', KEY_FINAL_VERDICT]),
+        "Contents": df_summary_json,
+        "EntryContext": {'DBotPredictURLPhishing': df_summary_json}
+    }
+    if tag is not None:
+        return_entry["Tags"] = ['DBOT_URL_PHISHING_{}'.format(tag)]
+    demisto.results(return_entry)
+    return df_summary_json
+
+
+def return_detailed_summary(results, number_entries_to_return):
+    outputs = []
+    severity_list = [x.get('score') for x in results]
+    indice_descending_severity = np.argsort(-np.array(severity_list), kind='mergesort')
+    for i in range(min(number_entries_to_return, len(results))):
+        index = indice_descending_severity[i]
+        pred_json = results[index].get('pred_json')
+        url = results[index].get('url')
+        is_white_listed = results[index].get('is_white_listed')
+        output_rasterize = results[index].get('output_rasterize')
+        summary_json = return_entry_summary(pred_json, url, is_white_listed, output_rasterize)
+        outputs.append(summary_json)
+        outputs = [x for x in outputs if x]
+    return outputs
+
+
+def save_model_in_demisto(model):
+    encoded_model = base64.b64encode(dill.dumps(model))  # guardrails-disable-line
+    res = demisto.executeCommand('createMLModel', {'modelData': encoded_model.decode('utf-8'),
+                                                   'modelName': URL_PHISHING_MODEL_NAME,
+                                                   'modelLabels': [MALICIOUS_VERDICT, BENIGN_VERDICT],
+                                                   'modelOverride': 'true',
+                                                   'modelHidden': True,
+                                                   'modelType': 'url_phishing',
+                                                   'modelExtraInfo': {
+                                                       OOB_MAJOR_VERSION_INFO_KEY: model.major,
+                                                       OOB_MINOR_VERSION_INFO_KEY: model.minor}})
+    if is_error(res):
```

Comment:
@erezh31 currently the model appears in the ML models page, and it seems like dev6 is updated, we need to add another config? 
![image](https://user-images.githubusercontent.com/57318201/135427357-3c97f3ac-75a4-46aa-8b7f-bfa7f0f70e15.png)

![image](https://user-images.githubusercontent.com/57318201/135427502-144a92ea-8511-4666-be36-f1fe65d69f8d.png)